docs/en_US/hpo_advanced.rst

```diff
@@ -0,0 +1,9 @@
+Advanced Features
+=================
+
+..  toctree::
+    Enable Multi-phase <AdvancedFeature/MultiPhase>
```

Comment:
suggest putting the multi-phase in last.

Reply:
@jodiemurphy13